languages/en.ini

```diff
@@ -382,6 +382,9 @@ fav_list_delete_fail = "Sorry, an error has occurred. Your list was not deleted.
 Favorites = "Saved Items"
 Fee = "Fee"
 Feedback = "Feedback"
+feedback_email = "Email"
```

Comment:
Looks like a couple of these strings duplicate strings from elsewhere in the language file. I have no objection to creating feedback-specific ones to aid customization, but we might as well pre-populate all the language files with reasonable defaults by copying the existing strings. There's a command line tool `php $VUFIND_HOME/public/index.php language copystring [source] [destination]` that can automate that work. I'm happy to take care of that step if you prefer!

Reply:
Fixed